servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/HeadersPropagatorGetter.java

```diff
@@ -67,4 +70,11 @@ public String get(@Nullable HttpHeaders carrier, final String key) {
         final CharSequence value = carrier.get(key);
         return value == null ? null : value.toString();
     }
+
+    static List<String> getHeadersValue(String name, HttpHeaders headers) {
+        Iterable<? extends CharSequence> value = headers.values(name);
+        return StreamSupport.stream(value.spliterator(), false)
```

Comment:
I was not sure on the best approach there, I wanted to avoid streams but I was skeptical on the amount of code necessary now I can see the benefits outweighs the short code.